packages/material-ui-lab/package.json

```diff
@@ -31,7 +31,7 @@
     "prebuild": "rimraf build",
     "release": "yarn build && npm publish build --tag next",
     "test": "cd ../../ && cross-env NODE_ENV=test mocha 'packages/material-ui-lab/**/*.test.js' --exclude '**/node_modules/**'",
```

Comment:
```suggestion
    "test": "cd ../../ && cross-env NODE_ENV=test mocha 'packages/material-ui-lab/**/*.test.{js,tsx}' --exclude '**/node_modules/**'",
```

Reply:
Discovered in https://github.com/mui-org/core-plus-pickers/pull/5#pullrequestreview-468008746